tests/common/integration_test.go

```diff
@@ -65,5 +65,23 @@ func WithUnixClient() config.ClusterOption {
 }
 
 func WithTCPClient() config.ClusterOption {
-	return integration.WithTCPClient()
+	return func(c *config.ClusterConfig) {
+		ctx := ensureIntegrationClusterContext(c)
```

Comment:
not sure I follow your change here. could you explain why it copies function from `integration.WithTCPClient()`? thanks

Reply:
+1
I think `return integration.WithTCPClient()` is ok.